.travis.yml

```diff
@@ -24,10 +24,22 @@ before_install:
   - rm -rf "$HOMEBREW_TAP_DIR"
   - ln -s "$PWD" "$HOMEBREW_TAP_DIR"
   - export HOMEBREW_DEVELOPER="1"
+  - export HOMEBREW_BINTRAY_USER="davidchall"
+  - export TAP_BOTTLE_DOMAIN="https://dl.bintray.com/davidchall"
   - ulimit -n 1024
 
 script:
-  - brew test-bot
+  - brew tap davidchall/test-bot
```

Comment:
Would love you to submit a PR for some of these changes upstream 😁

Reply:
I'll make sure to open a PR sometime soon.